doc/introduction/measurements.rst

```diff
@@ -154,6 +154,71 @@ so corresponds to a :math:`99.75\%` probability of measuring
 state :math:`|00\rangle`, and a :math:`0.25\%` probability of
 measuring state :math:`|01\rangle`.
 
+Mid-circuit measurements and conditional operations
+---------------------------------------------------
+
+PennyLane allows specifying measurements in the middle of the circuit.
+Operations can then be conditioned on the measurement outcome of such
+mid-circuit measurements:
+
+.. code-block:: python
+
+    def my_quantum_function(x, y):
+        qml.RY(x, wires=1)
```

Comment:
```suggestion
        qml.Hadamard(wires=0)
        qml.RY(x, wires=1)
```
Optional - this is very picky but the CNOT wouldn't do anything as it currently is shown.

Reply:
Fair point! I've changed to have the RY on 0.